### ISSUE TITLE
3D View - Armature - Pose Mode - New Select Grouped methods

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Manual#492]
![image](https://github.com/Bforartists/Manual/assets/35357409/0e3c67bb-bc98-4354-8ec8-38099610ef35)

Documentation needs updating (no more groups, but now collections)

And also with bone colour

<details>
<summary>Details</summary>

SHA-1: e41fbfd6e93913425d8a4b184f0bf2484fafd4f6

* Anim: add 'select grouped: bone collection' pose mode operator

Add 'Bone Collection' to the 'Select Grouped' operator in pose mode.
This removes the 'Same Layer' option, and replaces it with 'Same Bone
Collection'. It selects all the bones that are in the same collections
as the active bone.

When there are multiple armatures in pose mode, the bone collections are
matched by name.

</details>

<details>
<summary>Details</summary>

SHA-1: c230c5262612064d5cc3d6af38052a70796b1be3

* Anim: implement 'Select Grouped' by bone color

The pose mode 'Select Grouped' operator can now select by bone color.
This compares either the theme palette index, or the full custom color.
In other words, if custom colors are visually very similar but binary
not equal, they will be treated as different.

The last bytes of the custom colors (so `solid[3]`, `active[3]`, and
`selected[3]`) aren't used for these comparisons, as the values are
inconsistently either 0 or 255 depending on how the color was set. There
is no way to influence this with the color picker in the GUI, and AFAIK
the alpha channel isn't used for drawing either.

</details>

**Comment by ReinerBforartists** [comment]
Manual is already up to date. Task is invalid.
